static/index.html

```diff
@@ -6,9 +6,15 @@
     <title>Index</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="css/main.css" />
-    <script src="js/main.js"></script>
+
 </head>
 <body>
-    
+    <h1>Connected Modules</h1>
+    <p id="cTable">
+    </p>
```

Comment:
Don't use the p tag to store stuff. Use [div](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div) instead.

Reply:
Thanks for tip.